packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -24,7 +23,7 @@ context.setGlobalContextManager(contextManager);
 // DEBUGGING SETUP - setting the OpenTelemetry's internal diagnostic handler here to run when debug is enabled
 if (
   process.env.DEBUG &&
-  process.env.DEBUG === 'plugin:publish-metrics:open-telemetry'
+  process.env.DEBUG.includes('plugin:publish-metrics:')
```

Comment:
Setting OTels internal logger to run when `DEBUG` is set for any reporter, not just `open-telemetry`